concepticondata/conceptlists/Gerardi-2022-705.tsv

```diff
@@ -0,0 +1,706 @@
+Gerardi-2022-705-375	379	PATH OR ROAD	MOTION	2457	PATH OR ROAD
+Gerardi-2022-705-376	380	PATTERN	THE_PHYSICAL_WORLD	2988	PATTERN
+Gerardi-2022-705-377	381	PAU-MULATO	NA		
+Gerardi-2022-705-378	382	PAXIUBA	NA		
+Gerardi-2022-705-379	383	PAXIUBA (genus Irartea)	NA		
+Gerardi-2022-705-380	384	PAXIUBA-BARRIGUDA	NA		
+Gerardi-2022-705-381	385	PEACH PALM	AGRICULTURE_AND_VEGETATION	3919	PEACH PALM
+Gerardi-2022-705-382	386	PECCARY	ANIMALS	2390	PECCARY
+Gerardi-2022-705-383	387	PEEL	FOOD_AND_DRINK	275	PEEL
+Gerardi-2022-705-384	388	PENIS	THE_BODY	1222	PENIS
+Gerardi-2022-705-385	389	PEPPER	FOOD_AND_DRINK	1385	PEPPER
+Gerardi-2022-705-386	390	PERSPIRE OR SWEAT	THE_BODY	2458	PERSPIRE OR SWEAT
+Gerardi-2022-705-387	391	PESTLE	FOOD_AND_DRINK	225	PESTLE
+Gerardi-2022-705-388	392	PIAU	NA		
+Gerardi-2022-705-389	393	PICK	NA	7	GATHER
+Gerardi-2022-705-390	394	PICK UP	SPATIAL_RELATIONS	371	PICK UP
+Gerardi-2022-705-391	395	PIECE	QUANTITY	594	PIECE
+Gerardi-2022-705-392	396	PIED-CRESTED OROPENDOLA	NA		
+Gerardi-2022-705-393	397	PIERCE	BASIC_ACTIONS_AND_TECHNOLOGY	398	PIERCE
+Gerardi-2022-705-394	398	PINEAPPLE	AGRICULTURE_AND_VEGETATION	2440	PINEAPPLE
+Gerardi-2022-705-395	399	PIRANHA	ANIMALS	317	PIRANHA
+Gerardi-2022-705-396	400	PIRARUCU	NA		
+Gerardi-2022-705-397	401	PISS	THE_BODY	592	PISS
+Gerardi-2022-705-398	402	PIUM (simuliidae)	ANIMALS	3937	PIUM (SIMULIIDAE)
+Gerardi-2022-705-399	403	PLACE (POSITION)	SPATIAL_RELATIONS	723	PLACE (POSITION)
+Gerardi-2022-705-400	404	PLAY	EMOTIONS_AND_VALUES	1413	PLAY
+Gerardi-2022-705-401	405	POISON	THE_BODY	655	POISON
+Gerardi-2022-705-402	406	POLISH (SOMETHING)	BASIC_ACTIONS_AND_TECHNOLOGY	3659	POLISH (SOMETHING)
+Gerardi-2022-705-403	407	PORCUPINE	ANIMALS	2510	PORCUPINE
+Gerardi-2022-705-404	408	POT	FOOD_AND_DRINK	1462	POT
+Gerardi-2022-705-405	409	POUND TO FLATTEN	NA		
+Gerardi-2022-705-406	410	PRECIOUS	EMOTIONS_AND_VALUES	3002	PRECIOUS
+Gerardi-2022-705-407	411	PREGNANT	THE_BODY	1123	PREGNANT
+Gerardi-2022-705-408	412	PRETEND	COGNITION	3631	PRETEND
+Gerardi-2022-705-409	413	PROCREATE	NA		
+Gerardi-2022-705-410	414	PULL	BASIC_ACTIONS_AND_TECHNOLOGY	1455	PULL
+Gerardi-2022-705-411	415	PULL OFF (SKIN)	BASIC_ACTIONS_AND_TECHNOLOGY	291	PULL OFF (SKIN)
+Gerardi-2022-705-412	416	PULL OUT	BASIC_ACTIONS_AND_TECHNOLOGY	2325	PULL OUT
+Gerardi-2022-705-413	417	PUMA	ANIMALS	1891	PUMA
+Gerardi-2022-705-414	418	PUMPKIN	AGRICULTURE_AND_VEGETATION	864	PUMPKIN
+Gerardi-2022-705-415	419	PURPLE BANANA	NA		
+Gerardi-2022-705-416	420	PUSH	MOTION	1452	PUSH
+Gerardi-2022-705-417	421	PUT	SPATIAL_RELATIONS	998	PUT
+Gerardi-2022-705-418	422	RAIN (PRECIPITATION)	THE_PHYSICAL_WORLD	658	RAIN (PRECIPITATION)
+Gerardi-2022-705-419	423	RAINBOW	THE_PHYSICAL_WORLD	1733	RAINBOW
+Gerardi-2022-705-420	424	RAT	ANIMALS	1490	RAT
+Gerardi-2022-705-421	425	RED	SENSE_PERCEPTION	156	RED
+Gerardi-2022-705-422	426	RED-AND-GREEN MACAW (Ara chloropterus)	ANIMALS	3938	RED-AND-GREEN MACAW (Ara chloropterus)
```

Comment:
My last PR, did it not remove the (Ara chloperus)?

Reply:
Seems I did not push it, but it would be nice to quickly also change this Concepticon gloss, as we have removed all other glosses with lower case now.